features/admin.alternative-login-identifier.v1/pages/alternative-login-identifier-page.tsx

```diff
@@ -311,6 +323,56 @@ const AlternativeLoginIdentifierInterface: FunctionComponent<AlternativeLoginIde
         }
     };
 
+    const handleRevertSuccess = () => {
+        dispatch(
+            addAlert({
+                description: t(
+                    "governanceConnectors:notifications.revertConnector.success.description"),
+                level: AlertLevels.SUCCESS,
+                message: t(
+                    "governanceConnectors:notifications.revertConnector.success.message"
+                )
+            })
+        );
+    };
+
+    const handleRevertError = () => {
+        dispatch(
+            addAlert({
+                description: t(
+                    "governanceConnectors:notifications.revertConnector.error.description"
+                ),
+                level: AlertLevels.ERROR,
+                message: t(
+                    "governanceConnectors:notifications.revertConnector.error.message"
+                )
+            })
+        );
+    };
+
+    const onConfigRevert = () => {
+        setIsSubmitting(true);
+        const revertRequest: RevertGovernanceConnectorConfigInterface = {
+            properties: []
+        };
+
+        connector?.properties?.forEach((property: ConnectorPropertyInterface) => {
+            revertRequest.properties.push(property.name);
+        });
```

Comment:
We could do the assign during declaration using `map`

```suggestion
        const revertRequest: RevertGovernanceConnectorConfigInterface = {
            properties: connector?.properties?.forEach((property: ConnectorPropertyInterface) => property.name)
        };
```